### DISCUSSION TITLE
State badge should not make entity pictures round for camera, media player

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#11248] [Other feature requests]
## The request

The `<state-badge>` component will render a badge for a state. If a state has an entity picture, we're rendering that picture but in a circle. 

This is bad for camera and media player where the picture tells either what's currently visible (the whole point of a camera) or album art for a media player.

I think that we should explore changing 50% to 2px radius. It's probably better to just do it for all things that have an entity picture, bu we could also just limit it to media player/camera.

![Screen Shot 2020-03-30 at 16 51 57](https://user-images.githubusercontent.com/1444314/77972705-ffcd3e80-72a6-11ea-90dd-d66aac6042e8.png)

![Screen Shot 2020-03-30 at 16 52 19](https://user-images.githubusercontent.com/1444314/77972709-0360c580-72a7-11ea-8fac-79346971d5d9.png)

![image](https://user-images.githubusercontent.com/1444314/77972776-3014dd00-72a7-11ea-9cb6-2ffeb24eeb71.png)

![image](https://user-images.githubusercontent.com/1444314/77972794-3c993580-72a7-11ea-8906-2e7e704ada00.png)

It's also used in entities cards for rows.

**Comment by zsarnett** [comment]
I am not sure I agree. I don't think making them square gives you any more real detail at that small of an image. Circle looks better IMO

**Comment by matthiasdebaat** [comment]
I'm always a bit confused by the circle and think we should give it sharp edges. This gives it a more recognizable shape.
